### PR TITLE
Add product assets and update README

### PR DESCRIPTION
**Pull request by jinhojang6**
1. Add product assets which were missed at https://statusnetwork.com/press-kit/status/

2. Update README for testing locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+
+1. Open Terminal and navigate to the project root directory,
+
+2. Run `yarn install`,
+
+3. We share common elements across our sites using submodules, so you'll need to get those too:
+
+    ```bash
+    git submodule update --init --recursive
+    cd themes/navy/layout/partial/shared-partials
```
Comment (jakubgs) [inline README.md]: I don't think this is used anymore...

I don't have `themes/navy/layout/partial/shared-partials` and building the site works fine. We should remove `.gitmodules` and other references to submodules.

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+    git checkout master
+    git pull
+    cd ../../../../../
+    ```
+
+4. Run `./node_modules/.bin/gulp build`
```
Comment (jakubgs) [inline README.md]: We should make this into a `package.json` script, and upgrade to Gulp 4. I guess I can do it in a separate PR.

File: README.md (hunk 1 above)
Comment (jinhojang6) [reply README.md]: @jakubgs Sorry, forgot to remove the No.3. I've removed these unnecessary lines.

File: README.md (hunk 2 above)
Comment (jinhojang6) [reply README.md]: Thanks! I used this old gulp command to build the website

**Review by jakubgs — APPROVED** [review]

**Comment by jakubgs** [comment]
Please squash and sign the commits.

**Comment by jinhojang6** [comment]
@jakubgs  Done https://github.com/status-im/status-network-site/pull/11/commits

**Comment by jinhojang6** [comment]
@jakubgs Can we rebase the master branch to develop? This commit was not applied yet.

**Comment by jakubgs** [comment]
@jinhojang6 I've given you admin on this repo so you can do it yourself.

**Pull request merged**